app/src/main/res/drawable/splashscreen.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:drawable="@color/orange_hogwarts"/>
+    <item>
+        <bitmap android:src="@drawable/ic_hogwarts_logo"
+                android:gravity="center"/>
+    </item>
+</layer-list>
```

Comment:
add an empty line at the end of the file to avoid this error

Reply:
done